lib/streamlit/testing/v1/element_tree.py

```diff
@@ -11,6 +11,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+# ruff: noqa: S101
+
```

Comment:
Could we add comment/note that this is exempt as non-production code and/or a TODO?

Reply:
Yep, its fine here since app testing logic is supposed to be used in unit testing context. Added a comment 👍